app/src/main/java/com/example/harvestfresh/fragments/EventsFragment.java

```diff
@@ -125,20 +126,18 @@ private void goPlace(LatLng searchLatLng, String placeName) {
     private void placeMarkers(ParseGeoPoint userLocation) {
         ParseQuery<StoreFront> query = ParseQuery.getQuery(StoreFront.class);
         query.include(StoreFront.KEY_NAME);
-        query.setLimit(20);
+        query.setLimit(STORE_LIMIT);
 
         query.findInBackground(new FindCallback<StoreFront>() {
             @Override
             public void done(List<StoreFront> stores, ParseException e) {
                 if (e != null) {
-                    Log.e(TAG, ERROR_MESSAGE, e);
                     return;
                 }
                 for (StoreFront store : stores) {
                     LatLng newLocation = new LatLng(store.getLocation().getLatitude(), store.getLocation().getLongitude());
                     ParseGeoPoint storeMarker = new ParseGeoPoint(newLocation.latitude, newLocation.longitude);
                     double milesDistance = Math.round(userLocation.distanceInMilesTo(storeMarker));
-                    Log.d(TAG, MILES_AWAY + MILES_AWAY);
                     MarkerOptions newMarker = new MarkerOptions().position(newLocation).title(store.getName()).snippet(Double.toString(milesDistance) + MILES_AWAY).icon(BitmapDescriptorFactory.fromResource(R.drawable.mapsicon));
                     markerMap.addMarker(newMarker);
                 }
```

Comment:
Breaking up 152 like we talked about, and on 153, put each param on its on line like this:
getActivity(),
new String[] {Manifest.permission.Access_fine_location},
REQUEST_CODE

It makes it easier if each line is one atomic parameter for comparing against arg lists later.